weac/components/config.py

```
@@ -0,0 +1,39 @@
+In general, the configuration should only be changed by the developers and is
+static for the users with the most stable configuration.
+
+We utilize the pydantic library to define the configuration.
+
+Pydantic syntax is for a field:
+field_name: type = Field(..., gt=0, description="Description")
+- typing, default value, conditions, description
+"""
+
+import logging
+from typing import Literal
```

Ruff (0.12.2)

`typing.Literal` imported but unused

Remove unused import: `typing.Literal`

(F401)